### PR TITLE
Clockify (Independent Publisher)

### PR DESCRIPTION
**Pull request by iddycol**
---

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image

image

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thank you, Dr. Adrian, for submitting your first Independent Publisher connector! Our certification engineer was out for 2 weeks for a family emergency, so there will be a 1.5-2 week delay with reviewing your PR. We will let you know as soon as we review it. Apologies for the inconvenience!

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by nk-gears** [comment]
> Dear partner,
> 
> We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner.
> 
> Please let us know if you have any questions in the meantime.

@Amjed-Ayoub, I am a co-author for this Connector. Could you please let us know when this will be reviewed ?

**Comment by Amjed-Ayoub** [comment]
Hi @nk-gears, thank you for reaching out. 
Please expect progress within next week.
Thank you for your patience.

File: independent-publisher-connectors/Clockify/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+            "name": "archived",
+            "required": false,
+            "type": "boolean",
+            "x-ms-summary": "Archived",
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
```
[the remark this answers is not in the document]
Comment (nk-gears) [reply independent-publisher-connectors/Clockify/apiDefinition.swagger.json]: @v-eisljami , Could you please review now. Requested changes implemented

File: independent-publisher-connectors/Clockify/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+To use this connector, you need the following
+
+- A Microsoft Power Apps or Power Automate plan with custom connector feature
+- A Clockify account. Signup here : https://clockify.me/
+
+## Obtaining Credentials
+- Configure API Key here : https://clockify.me/user/settings
+
+
+### Supported Actions
```
Comment (v-eisljami) [inline independent-publisher-connectors/Clockify/README.md:18]: Could you please fix it to be the same as other headers? Also, the file needs to follow our template file, this section should be called Supported Operations. You can follow the official template on this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md).

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hi @iddycol, I hope you are doing well. I want to follow up with you and let you know that your connector is in production deployment. this process will complete within 4-6 weeks.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @iddycol, I hope you are doing well. This is to inform you that your connector is now fully deployed, and in documentation deployment now. The process will be complete by the end of the week.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @iddycol Congratulations! The deployment of your connector and documentation page is now complete. We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thank you very much

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**